ansible/roles/tasks-systemd/tasks/main.yml

```diff
@@ -79,10 +79,15 @@
 
       [container.secrets]
       # these are *host* paths, this is podman-remote
+      # FIXME: Split the upload/download secrets
       image-upload=[
           '--volume=/var/lib/cockpit-secrets/tasks/s3-keys/:/run/secrets/s3-keys:ro',
           '--env=COCKPIT_S3_KEY_DIR=/run/secrets/s3-keys',
       ]
+      image-download=[
```

Comment:
At first I was a bit confused, as alternative we could have `s3-bucket-token`. But this might be a bit too much of a bikeshed :)

Reply:
The point is that we eventually want to have *different* tokens for reading (enough for tests) and writing (for image refreshes). So we should name them by purpose. Also, S3 is an implementation detail at this point.